.github/workflows/neon_workflow.yml

```
@@ -27,72 +26,11 @@ jobs:
-  # You can uncomment the lines below to enable the schema diff action.
-      - name: Post Schema Diff Comment to PR
-        uses: neondatabase/schema-diff-action@v1
-        with:
-          project_id: ${{ vars.NEON_PROJECT_ID }}
-          compare_branch: preview/pr-${{ github.event.number }}-${{ needs.setup.outputs.branch }}
-          api_key: ${{ secrets.NEON_API_KEY }}
-
   create_neon_branch:
     name: Create Neon Branch
     outputs:
       db_url: ${{ steps.create_neon_branch_encode.outputs.db_url }}
```

actionlint (1.7.7)

property "create_neon_branch_encode" is not defined in object type {create_neon_branch: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)